.github/workflows/ci-tests.yml

```diff
@@ -1,31 +1,73 @@
 name: "ci tests"
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - 'README.md'
+      - 'ARCHITECTURE.md'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'README.md'
+      - 'ARCHITECTURE.md'
 
 jobs:
   test-tauri:
     strategy:
       fail-fast: false
       matrix:
-        platform: [macos-latest, ubuntu-latest, windows-latest]
+        os:
+          - ubuntu-latest
```

Comment:
Since we specify concrete versions, let's use `ubuntu-20.04` as well. Also maybe worth switching to macos-11 as the latest release, WDYT?

Reply:
I agree. If we are to follow concrete versions, changing it to `ubuntu-20.04` sounds good. However, keeping everything as `latest` is another idea. And I thought, since this will be a GUI, it may be good to know when there is a new bug emerges because of the OS's. My vote is actually on keeping everything as latest.